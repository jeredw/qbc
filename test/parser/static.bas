static wowzas() as string * 42
static i,j,k,a()
static a$,b$(),x,y,z
