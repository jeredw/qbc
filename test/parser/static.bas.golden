(program 
  (statement 
    (scope_statement static 
      (scope_variable 
        (untyped_id wowzas) 
        (array_declaration 
          ( )) as 
        (type_name 
          (fixed_string string * 42))))) \n 
  (statement 
    (scope_statement static 
      (scope_variable i) , 
      (scope_variable j) , 
      (scope_variable k) , 
      (scope_variable a 
        (array_declaration 
          ( ))))) \n 
  (statement 
    (scope_statement static 
      (scope_variable a$) , 
      (scope_variable b$ 
        (array_declaration 
          ( ))) , 
      (scope_variable x) , 
      (scope_variable y) , 
      (scope_variable z))) \n <EOF>)
